### DISCUSSION TITLE
Copilot got stuck in an infinite loop

### DISCUSSION THREAD

**Discussion by Earthcomputer** [community/community#7040] [Copilot Conversations]
Hi, copilot has been working great so far, saving me some time.
However I just got this rather amusing suggestion:
![Screenshot from 2021-10-30 10-20-19](https://user-images.githubusercontent.com/13084089/139527638-3d18cf68-1568-4636-8df7-0421e78b951e.png)
![Screenshot from 2021-10-30 10-32-00](https://user-images.githubusercontent.com/13084089/139527796-6e611217-5e60-429c-a2bf-0021cb6d567f.png)

When I tried accepting it, there was a cutoff at some point.

**Comment by nicovv44** [comment]
I believe you will need to provide your code for context...

Maybe your code was using recursion somewhere and it didn't like it.
